application/controllers/Validator.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Class to validate Email and Password.
+ */
+class Validator {
+  /**
+   * Function to validate email.
+   *
+   * @param string $email
+   *   User's email.
+   *
+   * @return bool
+   *  True if email is valid.
+   */
+  public function emailValidation(string $email): bool {
+    if (!filter_var($email, FILTER_VALIDATE_EMAIL)) {
```

Comment:
Why aren't we using the API for validate email ?
Infact there are email validatior packages available. Can we use them to validate email ?

Reply:
Free API call limit excedded and the Abstract api site is failed to create new user.
So avoided that.
I'll check about the email validatior packages.